packages/thumbprint-react/components/Type/index.tsx

```diff
@@ -56,51 +54,43 @@ export function Title({ children, size, className, headingLevel, dataTest, id })
     return React.createElement(elementName, props, children);
 }
 
-Title.propTypes = titlePropTypes;
-Title.defaultProps = titleDefaultProps;
-
-const textPropTypes = {
+interface TextPropTypes {
     /**
      * Text to render.
      */
-    children: PropTypes.node,
+    children?: React.ReactNode;
     /**
      * Size level of the text.
      */
-    size: PropTypes.oneOf([1, 2, 3]),
+    size?: 1 | 2 | 3;
     /**
      * Allows you to pass in additional styles to apply to the text. It is intentionally not
      * possible to overwrite `font-size` and `line-height`. If you need to do so, consider using
      * the ["Type" tokens](/tokens/#type) from Thumbprint Tokens or working with a designer to
      * use one of the styles on this page.
      */
-    className: PropTypes.string,
+    className?: string;
     /**
      * Name of the DOM element to render. Defaults to `p` for a paragraph tag.
      */
-    elementName: PropTypes.string,
+    elementName?: string;
```

Comment:
How about `keyof React.ReactHTML`?

See `React.createElement` type definition for reference:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L219-L222

Reply:
nice find. done